skimage/data/__init__.py

```diff
@@ -50,6 +53,172 @@
+        "data/bw_text.png": "308c2b09f8975a69b212e103b18520e8cbb7a4eccfce0f757836cd371f1b9094",
+        "data/bw_text_skeleton.npy": "9ff4fc23c6a01497d7987f14e3a97cbcc39cce54b2b3b7ee33b84c1b661d0ae1",
+        "data/_blobs_3d_fiji_skeleton.tif": "5182a2a94f240528985b8d15ec2aebbd5ca3c6b9214beff1eb6099c431e12b7b",
+        "data/checker_bilevel.png": "2e207e486545874a2a3e69ba653b28fdef923157be9017559540e65d1bcb8e28",
+        "restoration/tests/camera_rl.npy": "d219834415dc7094580abd975abb28bc7a6fb5ab83366e92c61ccffa66ca54fd",
+        "restoration/tests/camera_unsup.npy": "6d911fd0028ee78add8c416553097f15c6c4e59723ea32bd828f71269b6ea240",
+        "restoration/tests/camera_unsup2.npy": "30e81718f3cac0fc00d84518ca75a3c0fb9b246bb7748a9e20ec0b44da33734d",
+        "restoration/tests/camera_wiener.npy": "71e7cab739d6d145a288ec85dd235a62ff34442ccd1488b08139bc809850772b",
+        "feature/tests/data/OriginalX-130Y130.png": "bf24a06d99ae131c97e582ef5e1cd0c648a8dad0caab31281f3564045492811f",
+        "feature/tests/data/OriginalX130Y130.png": "7fdd4c06d504fec35ee0703bd7ed2c08830b075a74c8506bae4a70d682f5a2db",
+        "feature/tests/data/OriginalX75Y75.png": "c5cd58893c93140df02896df80b13ecf432f5c86eeaaf8fb311aec52a65c7016",
+        "feature/tests/data/TransformedX-130Y130.png": "1cda90ed69c921eb7605b73b76d141cf4ea03fb8ce3336445ca08080e40d7375",
+        "feature/tests/data/TransformedX130Y130.png": "bb10c6ae3f91a313b0ac543efdb7ca69c4b95e55674c65a88472a6c4f4692a25",
+        "feature/tests/data/TransformedX75Y75.png": "a1e9ead5f8e4a0f604271e1f9c50e89baf53f068f1d19fab2876af4938e695ea",
+    }
+)
+
+fetch = image_fetcher.fetch
+
+data_dir = image_fetcher.abspath
+
+# TODO: Load the requested readme by default
+
+def download_all(directory=None):
+    """Download all datasets for use with scikit-image offline.
+
+    Scikit-image datasets are no longer shipped with the library by default.
+    This allows us to use higher quality datasets, while keeping the
+    library download size small.
+
+    Call this function to download all sample images making them available
+    offline on your machine.
+
+    Parameters
+    ----------
+    directory: path-like, optional
+        The directory where the dataset should be stored.
+
+    Notes
+    -----
+    scikit-image will only search for images stored in the default directory.
+    Only specify the directory if you wish to download the images to your own
+    folder for a particular reason. You may access the location of the data
+    directory by inspecting the variable `skimage.data.data_dir`.
+    """
+
+    # Consider moving this kind of logic to Pooch
+    old_dir = image_fetcher.path
+    try:
+        if directory is not None:
+            image_fetcher.path = directory
+
+        for filename in image_fetcher.registry:
+            fetch(filename)
+    finally:
+        image_fetcher.path = old_dir
+
+
+def lbp_frontal_face_cascade_filename():
```

Comment:
this was moved in because i decided it didn't serve its own file for one function.